README.md

```diff
@@ -393,6 +393,7 @@ A curated list of awesome Java frameworks, libraries and software. Inspired by o
 *Libraries which provide general utility functions.*
 
 * [args4j](http://args4j.kohsuke.org/) - Command line arguments parser
+* [jcommander](http://jcommander.org/) - Command line arguments parser; its been around, active, and maintained by the creator since 2010.
```

Comment:
Please care about the spelling as it is used by the library, i.e. JCommander, isn't it?